lib/server.js

```diff
@@ -169,28 +172,40 @@ class S3Server {
             workerId: this.worker ? this.worker.id : undefined,
             workerPid: this.worker ? this.worker.process.pid : undefined,
         });
+        if (_config.clusters === 1) {
+            setTimeout(() => this.initiateStartup(logger.newRequestLogger()),
```

Comment:
Why not kill the process hard (`process.exit(1)`) and let kube restart the pod?

Reply:
I am actively making this change.